anapyzermodel.py

```diff
@@ -104,52 +94,33 @@ def set_log_type(self, log_type):
     def get_log_type(self):
         return self._log_type
 
+    # Setter for how the input file will be parsed
     def set_file_parse_mode(self, file_parse_mode):
         self._file_parse_mode = file_parse_mode
 
+    # Setter for how the input file will be parsed
     def get_file_parse_mode(self):
         return self._file_parse_mode
 
-    # Read the file
+    # Read the input file that is currently set in the model.
     def read_file(self):
```

Comment:
Any method which parses a log file should belong to the Model, not the Controller